blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently-ii.md

```diff
@@ -0,0 +1,71 @@
+---
+author: mihaiberq
+
+levels:
+
+  - basic
+
+  - medium
+
+type: normal
+
+category: must-know
+
+standards:
+  blockchain.identify-blockchain-applications.3: 10
+  blockchain.analyze-smart-contracts.0: 10
+
+aspects:
+
+  - introduction
+
+  - deep
+
+---
+# What Does Ethereum Do Differently? (Part 2)
+
+---
+## Content
+      	       

```

Comment:
how does this introduction sound? trying to make this insight more individual

cc @nem035
```suggestion
      	       
Ethereum couldn't be more different than Bitcoin. From using an account model, and having special transactions to solidity and token stardards. In this insight we'll take a look at some of those differences.

```

Reply:
I think "couldn't be more different", is a bit of an overstatement. I'd say Ethereum couldn't be more different than a pretzel or something but not Bitcoin 😆 

Anyhow, how about this?

```suggestion
      	       
Ethereum is different than Bitcoin in many ways. It has an account model, special transactions, a programming language (called Solidity) and token standards. In this insight we'll take a look at some of those differences.

```

Reply:
yeah, I went kinda hard when I said "couldn't be more different" :rofl: